app/extensions.js

```diff
@@ -54,7 +54,8 @@ let generateBraveManfiest = () => {
       'default-src': '\'self\'',
       'form-action': '\'none\'',
       'referrer': 'no-referrer',
-      'style-src': '\'self\' \'unsafe-inline\''
+      'style-src': '\'self\' \'unsafe-inline\'',
+      'img-src': '\'self\' data:'
```

Comment:
the loading issue actually has nothing to do with CSP, this was just required to allow the data urls